models/nc-news.models.js

```diff
@@ -212,3 +212,25 @@ exports.addCommentsOnArticle = (article_id, username, body) => {
       return rows[0];
     });
 };
+
+exports.removeCommentById = (commentId) => {
+  return db
+    .query(`SELECT * FROM comments`)
```

Comment:
again, you could use the WHERE clause here and check to rows length to dry this code up a bit